src/style.css

```diff
@@ -0,0 +1,154 @@
+body {
+  margin: 0;
+}
+
+.cards {
+  width: 200px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.cards__card {
+  /* width: 100%;
+  height: 50%; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.cards__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  margin-top: 32px;
+}
+
+.cards__description {
+  /* width: 100%;
+  height: 50%; */
+  display: flex;
+  justify-content: center;
+}
+
+.cards__content {
+  width: 166px;
+}
+
+.cards__title {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.cards__code {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.cards__rang {
+  height: 16px;
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+}
+
+.cards__feedback {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.cards__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.cards__star:last-child {
+  margin-right: 0;
+}
+
+.cards__star_4 {
```

Comment:
if I use cards__star_3 class, everything should work correctly 
<img width="195" alt="Screenshot 2022-07-17 at 16 04 45" src="https://user-images.githubusercontent.com/47606379/179399690-b6db0bbf-d1bc-496d-8a11-b97d4461922e.png">